src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/TeamsController.php

```diff
@@ -143,6 +163,35 @@ public function manageTeamAction(Request $request, int $serviceId)
         ]);
     }
 
+    /**
+     * @Method({"POST"})
+     * @Route("/service/{serviceId}/sendInvite/{teamId}", name="team_send_invite")
+     * @Security("has_role('ROLE_ADMINISTRATOR')")
+     *
+     * @return RedirectResponse|Response
+     */
+    public function sendInviteAction(Request $request, int $serviceId, int $teamId)
+    {
+        $this->get('session')->getFlashBag()->clear();
+        $email = $request->get('email');
+        $role = $request->get('role');
+        $invite = [
+            'teamId' => $teamId,
+            'intendedRole' => $role,
+            'emails' => $this->createEmailsArray($email, $role),
+            'message' => $this->translator->trans('teams.create.invitationMessage'),
+            'language' => 'ENGLISH',
+        ];
```

Comment:
Again, consider changing this into a more testable and dependable solution using VO's or DTO's